ganga/GangaLHCb/Lib/RTHandlers/GaudiExecRTHandlers.py

```diff
@@ -166,10 +166,10 @@ def prepareCommand(app):
                                                 getName(opts_file))
 
     #Check if this was checked out with LbEnv or not
-    isLbEnv = False
-    with open(app.directory+'/Makefile', "r") as makefile:
-        if 'LbEnv' in makefile.read():
-            isLbEnv = True
+    isLbEnv = True
```

Comment:
This fixes #1897 . I am not sure this ever worked and by now LbEnv has been mandatory for so long it shouldn't be an issue to remove the check. It breaks the resubmission with a prepared application as it looks in the build directory, which may not exist anymore if the application has been submitted already.